apisix/discovery/nacos.lua

```diff
@@ -344,6 +365,18 @@ function _M.init_worker()
         error('invalid nacos configuration: ' .. err)
         return
     end
+
+    events = require("resty.worker.events")
+    events_list = events.event_list(
+            "discovery_nacos_update_application",
+            "updating"
```

Comment:
Need to add `workers(4);` to https://github.com/apache/apisix/blob/0d7fa659d557a83224879cd1bfa823deff606bf7/t/discovery/nacos.t#L23 to check if it works

Don't forget to update https://github.com/apache/apisix/blob/master/docs/en/latest/discovery/nacos.md#service-discovery-via-nacos